apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -92,9 +92,9 @@ function escape() {
 # to the test runner
 test_env="%(test_env)s"
```

Comment:
It does seem like it should just always come down as part of this substitution but perhaps an improvement for a later time in Bazel.

Reply:
yea i don't think we have it in the process that does the substitutions here since that's a build and not a test run, but idk